src/App.jsx

```diff
@@ -1,35 +1,27 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import "./App.css";
 
-function App() {
-  const [count, setCount] = useState(0)
+const todoList = [
+  { id: 1, title: "Drumming" },
+  { id: 2, title: "Coding" },
+  { id: 3, title: "Socializing" },
+];
 
+function App() {
   return (
     <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+      <h1>Todo List</h1>
+
+      <ul>
+        {todoList.map((item) => {
+          return (
+            <li key={item.id} style={{ listStyle: "none" }}>
```

Comment:
I'm curious about your usage of `listStyle: "none"` here. Is it just aesthetic preference? Curiosity aside, it's great to see you use inline CSS – it can come in handy as you develop the look and feel of your app.

Reply:
Yep the bullets were bothering me ha.